FortranCon2020-stdlib/README.md

```diff
@@ -0,0 +1,37 @@
+# FortranCon2020-stdlib
+
+**Authors**:
+[Balint Arady](https://github.com/aradi)
+[Izaak Beekman](https://github.com/zbeekman)
+[Neil Carlson](https://github.com/nncarlson)
+[Ondrej Certik](https://github.com/certik)
+[Pedro Costa](https://github.com/p-costa)
+[Milan Curcic](https://github.com/milancuric)
+[Pierre de Buyl](https://github.com/pdebuyl)
+[???](https://github.com/fiolj)
+[???](https://github.com/JHenneberg)
+[Michael Hirsch](https://github.com/scivision)
+[Ivan Pribec](https://github.com/ivan-pi)
+[???](https://github.com/sakamoti)
+[???](https://github.com/nshaffer)
+[Jeremie Vandenplas](https://github.com/jvdp1)
+[Ashwin Vishnu](https://github.com/ashwinvis)
+
+
+**Track**: Junior presentations (15 + 5 minutes discussion)
+
+**Presenter**: To be confirmed
+
+**Title**: Fortran Standard Library
+
+**Alternative title**: Fortran Standard Library - a community project
+
+**Abstract**:
+The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
```

Comment:
Maybe add a sentence like:
```suggestion
The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
The language can be extended with new intrinsic procedure and modules, but these must be formally standardized and then implemented by compiler vendors before becoming available to users.
```
Maybe add a sentence for context? Not sure about word limits.